kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaCommandDecoder.java

```diff
@@ -294,11 +303,16 @@ protected void writeAndFlushResponseToClient(Channel channel) {
                         response.getRequest().getHeader());
                 }
 
-                ByteBuf result = responseToByteBuf(response.getResponseFuture().get(), response.getRequest());
+                ByteBuf result = responseToByteBuf(response.getResponseFuture().get(30,
+                        TimeUnit.SECONDS), response.getRequest());
```

Comment:
Could you explain why use the hard coded 30 seconds here?

Reply:
reference to `request.timeout.ms` in kafka.